CHANGELOG.md

```diff
@@ -1,12 +1,14 @@
 # CHANGELOG for django-crispy-forms
 
 ## 2.0 (tbc)
-* Removed `|safe` filter from field and prepended/appended text. To retain the previous 
-  behaviour use [`mark_safe()`](https://docs.djangoproject.com/en/4.0/ref/utils/#django.utils.safestring.mark_safe) 
-  to mark the text as safe in your project. Refs #296. 
 
-* Removed uni-form template pack. Uni-Form specific classes previously added to every template pack e.g. `textInput` and are now removed.
-  If you require these classes then the previous behaviour can be restored by adding the following to [CRISPY_CLASS_CONVERTERS](https://django-crispy-forms.readthedocs.io/en/latest/crispy_tag_forms.html#change-crispy-forms-input-default-classes) in your settings file.
+# Major Changes and Migration Guide
+* Removed Bootstrap 2 template pack. Bootstrap 3 and 4 support is provided by the core crispy-forms package.
+  Support for Bootstrap 5 is provided by a 3rd party package under the `django-crispy-forms` organisation at 
+  [crispy-bootstrap5](https://github.com/django-crispy-forms/crispy-bootstrap5).
```

Comment:
Maybe this is a bridge too far for now, but is it worth taking this opportunity to remove all the template packs? 🤔

Reply:
Hmm yes. I've not been that brave as yet. In part I'm worried that is overly disruptive. For example we will break other packages too. Django-filter comes to mind. 

It would be nice if we could get drf/df on bootstrap5 first.

Likely I worry too much.

Reply:
> Django-filter comes to mind.

So is it **just**, "install this extra package and set the template pack accordingly" — If so we'll cope 
(I didn't actually think about it yet: can do if you like... 😜)

Reply:
Hmm yes maybe it's install this package and update your settings file.  

I think most of the effort is in setting up the infrastructure of a new package. Maybe 4x as we could do it for the deprecated packages too...

Let me have a look. 👀

😀

Reply:
Ok, so a little progress here. I've published a package for [Bootstrap2](https://pypi.org/project/crispy-forms-bootstrap2/) and have the Bootstrap4 pack almost ready. I need another look, but I think I'll pin that to version 2 of crispy-forms core. 

Next up Bootstrap3. 

Not sure I have the energy for `uni-form`.

Reply:
OK, this is all done now 🎉  (well, appart from uni-form)